site/docs/master/contributions/ceph-radosgateway.md

```diff
@@ -0,0 +1,238 @@
+    "op_mask": "read, write, delete",
+    "default_placement": "",
+    "default_storage_class": "",
+    "placement_tags": [],
+    "bucket_quota": {
+        "enabled": false,
+        "check_on_raw": false,
+        "max_size": -1,
+        "max_size_kb": 0,
+        "max_objects": -1
+    },
+    "user_quota": {
+        "enabled": false,
+        "check_on_raw": false,
+        "max_size": -1,
+        "max_size_kb": 0,
+        "max_objects": -1
+    },
+    "temp_url_keys": [],
+    "type": "rgw",
+    "mfa_ids": []
+}
+```
+
+Install s3cmd to create a bucket by CLI
+
+```
+[root@ceph-mon1 ~]# yum install s3cmd
+```
+
+Configure s3cmd to use my Rados GW endpoint
+
+```
+[root@ceph-mon1 ~]# s3cmd --configure
+Enter new values or accept defaults in brackets with Enter.
+Refer to user manual for detailed description of all options.
+Access key and Secret key are your identifiers for Amazon S3. Leave them empty for using the env variables.
+Access Key [AOTBA6CUYR4P2WD7Q7ZK]: 
+Secret Key [d4ZY0cmAQcsmviwcpshE0bjWfyT5RDUROUE0BmJ6]: 
+Default Region [US]: 
+Use "s3.amazonaws.com" for S3 Endpoint and not modify it to the target Amazon S3.
+S3 Endpoint [radosgw.local.lab:80]: 
+Use "%(bucket)s.s3.amazonaws.com" to the target Amazon S3. "%(bucket)s" and "%(location)s" vars can be used
+if the target S3 system supports dns based buckets.
+DNS-style bucket+hostname:port template for accessing a bucket [%(bucket)s.radosgw.local.lab]: radosgw.local.lab:80
+Encryption password is used to protect your files from reading
+by unauthorized persons while in transfer to S3
+Encryption password: 
+Path to GPG program [/usr/bin/gpg]: 
+When using secure HTTPS protocol all communication with Amazon S3
+servers is protected from 3rd party eavesdropping. This method is
+slower than plain HTTP, and can only be proxied with Python 2.7 or newer
+Use HTTPS protocol [No]: 
+On some networks all internet access must go through a HTTP proxy.
+Try setting it here if you can't connect to S3 directly
+HTTP Proxy server name: 
+New settings:
+  Access Key: AOTBA6CUYR4P2WD7Q7ZK
+  Secret Key: d4ZY0cmAQcsmviwcpshE0bjWfyT5RDUROUE0BmJ6
+  Default Region: US
+  S3 Endpoint: radosgw.local.lab:80
+  DNS-style bucket+hostname:port template for accessing a bucket: radosgw.local.lab:80
+  Encryption password: 
+  Path to GPG program: /usr/bin/gpg
+  Use HTTPS protocol: False
+  HTTP Proxy server name: 
+  HTTP Proxy server port: 0
+Test access with supplied credentials? [Y/n] y
+Please wait, attempting to list all buckets...
+Success. Your access key and secret key worked fine :-)
+Now verifying that encryption works...
+Not configured. Never mind.
+Save settings? [y/N] y
+Configuration saved to '/root/.s3cfg'
+```
+
+Create a bucket for Velero
+```
+[root@ceph-mon1 ~]# s3cmd mb s3://velero
+Bucket 's3://velero/' created
+```
+
+2. Velero Install
+
+CLI Download
+```
+$ wget https://github.com/vmware-tanzu/velero/releases/download/v1.2.0/velero-v1.2.0-linux-amd64.tar.gz
```

Comment:
It'd be nice to update this to use velero v1.4.0 if possible

Reply:
Sure, I will update this , let me test in my environment and confirm for you!

Reply:
@fabioabreureis Were you able to get this updated?

Reply:
Hi ! My appologies for my reploy so late. Yes, I will validate for velero 1.4 as recommended and update the document. 

Thanks.